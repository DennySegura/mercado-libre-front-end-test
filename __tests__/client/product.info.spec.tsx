import 'jest';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProductInfo from '../../src/app/components/product-info'

describe('Test suites for react components', () => {
  it('It should render <ProductInfo />', () => {
    const data = {
      title: 'Smart Tv 4k 50  Samsung Un50mu6100',
      condition: 'new',
      sold_quantity: 500,
      price: {
        decimals: 0,
        amount: 21999,
        currency: 'ARS'
      }
    }
    const _product_info = renderer.create(<ProductInfo { ...data }/>);
    let product_info_tree = _product_info.toJSON();
    expect(product_info_tree).toMatchSnapshot();
  });
});
