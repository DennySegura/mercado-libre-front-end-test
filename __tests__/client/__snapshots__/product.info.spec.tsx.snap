// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suites for react components It should render <ProductInfo /> 1`] = `
<div
  className="column"
>
  <span
    className="anotation mb"
  >
    New - 500 sold
  </span>
  <span
    className="headline"
  >
    Smart Tv 4k 50  Samsung Un50mu6100
  </span>
  <span
    className="price--container mv3"
  >
    <span
      className="price--amount mr"
    >
      $
    </span>
    <span
      className="price--amount"
    >
      21.999
    </span>
    <span
      className="price--decimals"
    >
      00
    </span>
  </span>
</div>
`;
